ycms/cms/views/ward/ward_management_view.py

```diff
@@ -0,0 +1,94 @@
+from django.contrib import messages
+from django.shortcuts import redirect, render
+from django.utils.translation import gettext as _
+from django.views.generic import TemplateView
+
+from ...constants import job_types
+from ...forms import WardForm
+from ...models import User, Ward
+
+
+class ZentralView(TemplateView):
+    """
+    View to see all wards data and add a new one
+    """
+
+    template_name = "ward/ward_management.html"
+
+    def get(self, request, *args, **kwargs):
+        r"""
+        This function returns a list of all wards,
+        as well as a form for creating a new one
+
+        :param request: The current request
+        :type request: ~django.http.HttpRequest
+
+        :param \*args: The supplied arguments
+        :type \*args: list
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+
+
+        """
+
+        all_wards = Ward.objects.all()
+        ward_form = WardForm()
+
+        return render(
+            request,
+            self.template_name,
+            {
+                "all_wards": all_wards,
+                "wards": self._get_ward_info(all_wards),
+                "ward_form": ward_form,
+                # **super().get_context_data(**kwargs),
+            },
+        )
+
+    def _get_ward_info(self, wards):
+        ward_info = {}
+
+        ward_info["all_wards_count"] = wards.count()
+
+        ward_info["all_beds_count"] = ward_info["all_wards_count"] * 28
+
+        ward_info["all_occupied_beds"] = sum(ward.occupied_beds for ward in wards)
+        ward_info["all_available_beds"] = sum(ward.available_beds for ward in wards)
+
+        ward_info["all_doctors_count"] = User.objects.filter(
+            job_type=job_types.DOCTOR
+        ).count()
+
+        ward_info["all_nurses_count"] = User.objects.filter(
+            job_type=job_types.NURSE
+        ).count()
+
+        return ward_info
```

Comment:
```suggestion
    @staticmethod
    def _get_ward_info():
        wards = Ward.objects.all()
        return {
            "wards": wards,
            "wards_count": wards.count(),
            "beds_count": sum(ward.total_beds for ward in wards)
            "occupied_beds": sum(ward.occupied_beds for ward in wards),
            "available_beds": sum(ward.available_beds for ward in wards),
            "doctors_count": User.objects.filter(job_type=job_types.DOCTOR).count(),
            "nurses_count": User.objects.filter(job_type=job_types.NURSE).count(),
        }
```
This does the exact same thing (apart from the fix to `beds_count` that I have mentioned before), but is a bit more pythonic and nicer to read / add stuff later. I've also taken the liberty to remove the `all_` prefixes, since they are all in one place now :)